test/integration/partial.js

```diff
@@ -250,20 +264,7 @@ tape('carol acts as an intermediate between alice and bob', async (t) => {
       debouncePeriod: 1,
       partialReplication: {
         0: null,
-        1: {
-          subfeeds: [
-            { feedpurpose: 'mygame' },
-            {
-              feedpurpose: 'indexes',
-              subfeeds: [
-                {
-                  feedpurpose: 'index',
-                  $format: 'indexed',
-                },
-              ],
-            },
-          ],
-        },
+        1: [{ purpose: 'mygame' }, { purpose: 'index' }],
```

Comment:
great to see how much simpler this seems to be. The complexity of the feed management into groups is taken care of by the shards. Fingers crossed this works well in practice.